deploy_stack.arm64.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
Can you see if you can update deploy_stack.sh for @rgee0's latest change and incorporate that too? Perhaps we can avoid having to add this file.